README.md

```diff
@@ -1,16 +1,115 @@
-##Pepperdata Datasource
+# Grafana data source plugin template
 
+This template is a starting point for building a Data Source Plugin for Grafana.
 
-A work in progress plugin for using Pepperdata as a datasource. Currently only the test and query endpoints are supported.
+## What are Grafana data source plugins?
 
+Grafana supports a wide range of data sources, including Prometheus, MySQL, and even Datadog. There’s a good chance you can already visualize metrics from the systems you have set up. In some cases, though, you already have an in-house metrics solution that you’d like to add to your Grafana dashboards. Grafana Data Source Plugins enables integrating such solutions with Grafana.
 
-###Install
+## Getting started
 
-Put the /dist directory in your grafana plugins folder
 
+### Frontend
 
-###Configure
+1. Install dependencies
```

Comment:
might want to add some steps here on how we got this working for us like doing a brew install grafana, edit the ini, link the dir, and setup apikeys

Reply:
Ok added some steps for getting grafana set up. Will add info about apikeys in the src/README since it seems like that readme is more about the plugin itself and how to configure it, not plugin development.